packages/config/src/services/formDraftService.ts

```diff
@@ -86,28 +77,25 @@ export async function clearHearthElasticInfluxData(request: Hapi.Request) {
     throw Error(`Failed to delete elastic, influx data. ${err}`)
   }
 
-  hearthDBConn.db.listCollections().toArray((err, collections) => {
-    if (err) {
-      logger.error(`Error occured getting list of hearthDB collections: ${err}`)
-    } else {
-      Promise.all(
-        collections.map(async (collection) => {
-          if (Object.values(HearthCollectionsName).includes(collection.name)) {
-            try {
-              await hearthDBConn.dropCollection(collection.name)
-              logger.info(`Droped hearthDB collection :: ${collection.name} `)
-            } catch (err) {
-              logger.error(
-                `Error occured droping collection name (${collection.name}) : ${err}`
-              )
-            }
-          }
+  try {
+    Promise.all(
+      hearthCollectionList.map(async (collection) => {
+        const data = await fetchFHIR(`/${collection}`, {
+          Authorization: `Bearer ${token}`
+        })
+        data.entry.map(async (entry: fhir.BundleEntry) => {
+          await deleteFHIR(`/${collection}/${entry.resource?.id}`, {
+            Authorization: `Bearer ${token}`
+          })
+          logger.info(
+            `Deleting '${collection}' collection's resource id: ${entry.resource?.id}`
+          )
         })
-      ).then(() => {
-        logger.info('Successfully droped all listed hearthDB collections')
       })
-    }
-  })
+    )
+  } catch (err) {
+    throw Error(`Failed to delete hearth data. ${err}`)
+  }
```

Comment:
Remove this try-catch as it really only makes the error more difficult to handle. It's best to keep errors as-is as long as we can for easier debugging and error handling